### DISCUSSION TITLE
Not able to upload a file in Drive

### DISCUSSION THREAD

**Discussion by deependraPatel** [deta/cloud-docs#354] [Help] [answered]
I am using NodeJs for my project and trying to upload a new file in Drive.
But I am getting - "Something went wrong.", attaching a screenshot.

Any help is grateful. Thanks

![image](https://user-images.githubusercontent.com/18331661/181732421-ba82e48e-6bfc-44e6-be64-d82eb06f4642.png)

**Comment by xeust** [comment, marked as answer]
Hi @deependraPatel,

What kind of file are you trying to upload?

Would you mind sharing a code snippet of how you are using the SDK in your app?

**Reply by deependraPatel** [reply]
I am trying to upload image (jpg) file.

Attaching code snippet.

![image](https://user-images.githubusercontent.com/18331661/181776807-bfb6a575-9333-47e7-a497-4e599ae84a3f.png)

**Reply by w3bsite** [reply]
> I am trying to upload image (jpg) file.
> 
> Attaching code snippet.
> 
> ![image](https://user-images.githubusercontent.com/18331661/181776807-bfb6a575-9333-47e7-a497-4e599ae84a3f.png)

maybe Im wrong,  but it seems you are using nodemon and node start index.js,
are you aware about the deta cli and it's watch tool?

if you have deta cli instaled allready, you should login with 'deta login' and after that 'deta watch' , what it does is watch you code (like nodemon) and after every change (and save) deploys your code to deta servers, 
if you open your project in your deta dashboard you can find (and add) a domain that you can use to access every endpoint you are creating and is being deployed to deta servers,
so in short:

1-install deta cli
2-deta login
3-deta watch

your endpoint should be online now

Hope it helps

**Reply by deependraPatel** [reply]
Thanks. It really helps.
Really appreciate for your help.
